python/mxnet/module/base_module.py

```diff
@@ -80,14 +80,14 @@ class BaseModule(object):
     """The base class of a module.
 
     A module represents a computation component. Modules are designed so that
-    they can be thoughtof as a computation "machine". Each model can run forward,
+    they can be thought of as a computation "machine". Each model can run forward,
```

Comment:
Each model can run forward, backward, update its parameters, etc. -> A module can execute forward and backward passes and update parameters in a model.